### PR TITLE
Several fixes

### LINKED ISSUE #9585: [BUG] when you feed citizens food in bowls they eat the bowl
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [ ] Multi Player

### Minecraft Version

1.20.1

### MineColonies Version

1.20.1-1.1.240-release

### Structurize Version

not related to structurize

### Related Mods and their Versions

_No response_

### Current Behavior

the bowl disappears after bowl food is eaten

### Expected Behavior

the bowl is emptied

### Reproduction Steps

1. feed citizen bowl food (example: mushroom stew)

### Logs

https://gist.github.com/SonicTheHedgehog58987276/45e7a2d63c8b2e908237489024d47eca

### Anything else?

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments. i am affected
* Add a comment if you have any insights or background information that isn't already part of the conversation.

### LINKED ISSUE #9586: [BUG] Supply camp/ship error highlights remain after placement
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [X] Multi Player

### Minecraft Version

1.19.2

### MineColonies Version

321.beta

### Structurize Version

??

### Related Mods and their Versions

Unknown. Uncertain if it is 1.19.2 and 1.20.1, or either/or.

### Current Behavior

This is a just a heads up report for dev team
Many reports on main discord regarding highlights remaining after placing supply camps / ships. Started immediately after version .321 was released.
My advice has been to restart Minecraft and this has worked to remove them. Relog alone does not remove them.

### Expected Behavior

Highlights to disappear after placement

### Reproduction Steps

Place supply camp or ship
Observe highlights do not go away.

### Logs

Discord reports below

### Anything else?

Screenshots included on these discord reports below.
https://discord.com/channels/139070364159311872/1003402487417548801/1175996369018830951
https://discord.com/channels/139070364159311872/1125094832143077456/1176692042814988308
https://discord.com/channels/139070364159311872/1003402487417548801/1176307330191728680
https://discord.com/channels/139070364159311872/1003402520846139533/1176458553935003658
https://discord.com/channels/139070364159311872/1003402487417548801/1175876932739543100

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

### LINKED ISSUE #9587: [BUG] Teaching recipes to crafters does not fully work to update clipboards etc.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [ ] Multi Player

### Minecraft Version

1.20.1

### MineColonies Version

1.20.1-1.1.321-BETA

### Structurize Version

1.20.1-1.0.662-BETA

### Related Mods and their Versions

forge 47.2.0
blockui-1.20.1-1.0.125-BETA.jar
domum_ornamentum-1.20-1.0.147-BETA-universal.jar
multipiston-1.20-1.2.31-ALPHA.jar

### Current Behavior

teaching a recipe often doesn't make the 'ding' and doesn't remove the 'need' from the clipboard. The recipe is in fact taught
Happens for normal recipes and DO, at all crafters including like builder; canceling from clipboard, they figure it out
I'd estimate it happens 40% of the time or so; the rest of the time the normal ding occurs and the clipboard immediately updates

### Expected Behavior

Ding
immediate clipboard updates (for normal recipes at least)

### Reproduction Steps

Open crafter UI 
Use JEI to search and teach a recipe (for example some stone or wood slab)
Click Learn

### Logs

https://gist.github.com/Shadizar/3a4215d3b90083f92f966926f495d5f2

### Anything else?

Multiple persons reported in 1.20 discord (at least 2 others IIRC)

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #9585 
Closes #9587
Closes #9586

# Changes proposed in this pull request:
- Reduce noise volume to half of normal voice lines
- Fix issues with crafting recipe adding
- Fix jobs on stat page
- Fix way we handle container items.

[ x ] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java [hunk 1]
```diff
@@ -432,12 +432,17 @@ public boolean addRecipe(final IToken<?> token)
             final IRecipeStorage recipeStorage = IColonyManager.getInstance().getRecipeManager().getRecipes().get(token);
             if (recipeStorage != null)
             {
+                if (recipeStorage.getAlternateOutputs().isEmpty())
+                {
+                    //building.getColony().getRequestManager().onColonyUpdate(request -> request.getRequest() instanceof IDeliverable && ((IDeliverable) request.getRequest()).matches(recipeStorage.getPrimaryOutput()));
+                    //return true;
+                }
+
```
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java:440]: Empty if?

I may be missing something, but there doesn't seem to be an actual code change here (other than some whitespace changes)
I'm wondering how this is solving the clipboard issue

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java:440]: oopsy.

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java:440]: The fix is actually making the ItemStackUtils a boolean and not a Boolean. But I did forgot to re-comment this =D

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java:440]: I wanted to have this "quick option" as this is more performant for most things.

Comment (MotionlessTrain) [reply src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java:440]: Ah, I see. That part made it into another commit :smile:

**Review by someaddons — DISMISSED** [review]

File: src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java [hunk 2]
```diff
@@ -432,12 +432,17 @@ public boolean addRecipe(final IToken<?> token)
             final IRecipeStorage recipeStorage = IColonyManager.getInstance().getRecipeManager().getRecipes().get(token);
             if (recipeStorage != null)
             {
+                if (recipeStorage.getAlternateOutputs().isEmpty())
+                {
+                    building.getColony().getRequestManager().onColonyUpdate(request -> request.getRequest() instanceof IDeliverable && ((IDeliverable) request.getRequest()).matches(recipeStorage.getPrimaryOutput()));
```
Comment (uecasm) [inline src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java]: This should use a pattern cast, like the other branch does.

**Pull request merged**